lib/lighthouse_bgs/services/vet_record.rb

```diff
@@ -36,5 +36,20 @@ def update_birls_record(ssn:, poa_code:)
       )
       response.body[:update_birls_record_response][:return]
     end
+
+    # find CorpDB records with first and last name
+    def find_corporate_record(first_name:, last_name:)
+      response = request(
+        :find_corporate_record,
+        {
+          "ptcpntSearchPSNInput": {
+            "firstName": first_name,
+            "lastName": last_name
+          }
+        },
+        ssn
```

Comment:
I don't think we need to pass this

Reply:
if `mock_responses` is false it wont matter, but if its true, then this arg will be used to identify the mock data file to pull

Reply:
Also, there is no `ssn` arg in this method